packages/bruno-app/src/components/ResponsePane/ResponseClear/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { IconEraser } from '@tabler/icons';
+import { useDispatch } from 'react-redux';
+import StyledWrapper from './StyledWrapper';
+import { responseReceived } from 'providers/ReduxStore/slices/collections/index';
+
+const ResponseClear = ({ collection, item }) => {
+  const dispatch = useDispatch();
+  const response = item.response || {};
+
+  const clearResponse = () =>
+    dispatch(
+      responseReceived({
+        itemUid: item.uid,
+        collectionUid: collection.uid,
+        response: null
+      })
```

Comment:
I'm not sure it is proper way to call `responseReceived` method. Probably there should be new method for response clearing in `providers/ReduxStore/slices/collections/index.js`  (e.g. `responseCleared`). Also `responseReceived` method is setting `item.requestState` to `'received'` which I guess is not good in such case.

Reply:
Done!